node_package/src/createReactOutput.ts

```
@@ -26,6 +26,8 @@ export default function createReactOutput({
 }: CreateParams): CreateReactOutputResult {
   const { name, component, renderFunction } = componentObj;
 
+  const children = props?.children_html ? React.createElement('div', {dangerouslySetInnerHTML: {__html: props.children_html }}) : null;
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)